packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -0,0 +1,49 @@
+import type { ReactElement } from 'react';
+import { useState } from 'react';
+import styled from 'styled-components/native';
+import type { BaseInputProps } from './BaseInput';
+import getBaseInputStyles from './getBaseInputStyles';
+import type { Theme } from '~components/BladeProvider';
+
+export const StyledNativeBaseInput = styled.TextInput(
+  (props: BaseInputProps & { isFocussed: boolean; theme: Theme }) => ({
+    ...getBaseInputStyles({ theme: props.theme }),
+    backgroundColor: props.isFocussed
+      ? props.theme.colors.brand.primary[300]
+      : props.theme.colors.brand.gray[200],
+    borderBottomColor: props.isFocussed
+      ? props.theme.colors.brand.primary[500]
+      : props.theme.colors.brand.gray[400],
+  }),
+);
+
+export const StyledBaseInput = ({
+  label,
+  labelPosition = 'top',
+  placeholder,
+  type = 'text',
+  defaultValue,
+  name,
+  onChange,
+}: BaseInputProps): ReactElement => {
+  const [isFocussed, setIsFocussed] = useState(false);
```

Comment:
Not a nit 🙈: https://proofed.com/writing-tips/spelling-tips-focused-or-focussed/

```suggestion
  const [isFocused, setIsFocused] = useState(false);
```

Reply:
fixed in #632